features/step_definitions/mappings_import_assertion_steps.rb

```diff
@@ -11,6 +11,17 @@
   }
 end
 
+Then(/^I should see how many archive mappings will be created$/) do
+  steps %{
+    Then I should see "Create 3 new archives"
```

Comment:
This step's assertion duplicates part of the following step - is there a way of rewording the description of the latter so that it makes sense to use only that one?